s5.go

```diff
@@ -40,13 +41,27 @@ func run(c *cli.Context) error {
 	return exit(err)
 }
 
-func cipher(c *cli.Context) error {
+func readInput(c *cli.Context) (input string, err error) {
 	if c.NArg() != 1 {
-		cli.ShowSubcommandHelp(c)
-		return cli.NewExitError("", 1)
+		read, err := ioutil.ReadAll(os.Stdin)
+		if err != nil {
+			return "", exit(cli.NewExitError(err.Error(), 1))
+		}
+		input = string(read)
+	} else {
```

Comment:
what about the `c.NArg() > 1` case?

Reply:
absolutely! just reverted to the behaviour :)